docs/.vuepress/config.js

```diff
@@ -47,6 +47,7 @@ module.exports = {
             level: 999, // Disable original Plugin
         },
         extendMarkdown: (md) => {
+            md.use(require('markdown-it-attrs'));
```

Comment:
This plugin will break the TOC
![image](https://user-images.githubusercontent.com/11386903/189524355-3006f167-0255-42b6-bcd0-bd630ccd2227.png)

Reply:
OK